pages/hackathon/[slug]/register.tsx

```diff
@@ -1,28 +1,31 @@
 /* eslint-disable react/react-in-jsx-scope */
 import { useRouter } from "next/router"
-import axios from "axios"
+import axios, { AxiosError, AxiosResponse } from "axios"
```

Comment:
Can you change this to importing Axios from `utils/axios` to have a common Axios instance for the whole repo? And maybe export Axios types from the same?

Reply:
Done